vmmlib/enable_if.hpp

```diff
@@ -41,12 +41,17 @@
  */
 namespace vmml
```

Comment:
probably not anymore. It's c&p from boost enable_if, but now we should use C++11. Good catch!